<launch>

  <arg name="model" default="waffle_pi" doc="model type [burger, waffle, waffle_pi]"/>
  

  <param name="robot_description" command="$(find xacro)/xacro $(find turtlebot3_description)/urdf/turtlebot3_$(arg model).urdf.xacro" />

  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
    <param name="publish_frequency" type="double" value="30.0" />
  </node>
  
  <node name="fake_laser" pkg="era_gazebo" type="fake_laser" />

  <node pkg="tf" type="static_transform_publisher" name="transform_publisher_map" args="0 0 0 0 0 0 /map /odom 100"/>

   <node pkg="tf" type="static_transform_publisher" name="transform_publisher_odom" args="0 0 0 0 0 0 /odom /base_footprint 100"/>

  <node name="costmap_node" pkg="costmap_2d" type="costmap_2d_node" >
    <rosparam file="$(find era_gazebo)/launch/costmap_2d.yaml" command="load" ns="costmap" />
    <remap from="costmap_node/costmap/costmap" to="local_map"/>
  </node>


</launch>
